provider-example/src/hpke.rs

```diff
@@ -53,6 +59,10 @@ impl Debug for HpkeRs {
 }
 
 impl Hpke for HpkeRs {
+    fn suite(&self) -> HpkeSuite {
```

Comment:
Nit: `suite()` should go at the bottom (in the impls, too).

Reply:
Moved in the trait & impls.